releng testJavadocLogs test fails in I20220909-0040

See https://download.eclipse.org/eclipse/downloads/drops4/I20220909-0040/compilelogs/platform.doc.isv.javadoc.txt

```
../../../eclipse.platform.ui/bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/application/WorkbenchAdvisor.java:689: error: reference not found
	 * @see IWorkbenchWindowConfigurer#createMenuBar
	        ^
../../../eclipse.platform.ui/bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/application/WorkbenchAdvisor.java:690: error: reference not found
	 * @see IWorkbenchWindowConfigurer#createCoolBarControl
	        ^
../../../eclipse.platform.ui/bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/application/WorkbenchAdvisor.java:691: error: reference not found
	 * @see IWorkbenchWindowConfigurer#createStatusLineControl
	        ^
../../../eclipse.platform.ui/bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/application/WorkbenchAdvisor.java:692: error: reference not found
	 * @see IWorkbenchWindowConfigurer#createPageComposite
	        ^
../../../eclipse.platform.ui/bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/application/WorkbenchWindowAdvisor.java:272: error: reference not found
	 * @see IWorkbenchWindowConfigurer#createMenuBar
	        ^
../../../eclipse.platform.ui/bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/application/WorkbenchWindowAdvisor.java:273: error: reference not found
	 * @see IWorkbenchWindowConfigurer#createCoolBarControl
	        ^
../../../eclipse.platform.ui/bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/application/WorkbenchWindowAdvisor.java:274: error: reference not found
	 * @see IWorkbenchWindowConfigurer#createStatusLineControl
	        ^
../../../eclipse.platform.ui/bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/application/WorkbenchWindowAdvisor.java:275: error: reference not found
	 * @see IWorkbenchWindowConfigurer#createPageComposite
	        ^
8 errors
```

@vogella : this is coming from https://github.com/eclipse-platform/eclipse.platform.ui/pull/322

--- a/bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/application/WorkbenchAdvisor.java
+++ b/bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/application/WorkbenchAdvisor.java
@@ -686,10 +686,6 @@ public abstract class WorkbenchAdvisor {
 	 *
 	 * @param configurer the window configurer
 	 * @param shell      the window's shell
-	 * @see IWorkbenchWindowConfigurer#createMenuBar
-	 * @see IWorkbenchWindowConfigurer#createCoolBarControl
-	 * @see IWorkbenchWindowConfigurer#createStatusLineControl
-	 * @see IWorkbenchWindowConfigurer#createPageComposite
 	 *
 	 * @deprecated since 3.1, override
 	 *             {@link WorkbenchWindowAdvisor#createWindowContents(Shell)}

--- a/bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/application/WorkbenchWindowAdvisor.java
+++ b/bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/application/WorkbenchWindowAdvisor.java
@@ -269,10 +269,6 @@ public class WorkbenchWindowAdvisor {
 	 * </p>
 	 *
 	 * @param shell the window's shell
-	 * @see IWorkbenchWindowConfigurer#createMenuBar
-	 * @see IWorkbenchWindowConfigurer#createCoolBarControl
-	 * @see IWorkbenchWindowConfigurer#createStatusLineControl
-	 * @see IWorkbenchWindowConfigurer#createPageComposite
 	 * @deprecated This method is no longer used. Applications now define workbench
 	 *             window contents in their application model.
 	 */

